client/src/pages/welcome-grasshopper.js

```diff
@@ -0,0 +1,259 @@
+import React from 'react';
+import { Grid, Row, Col, Image } from '@freecodecamp/react-bootstrap';
+import FontAwesomeIcon from '@fortawesome/react-fontawesome';
+import {
+  faHtml5,
+  faCss3Alt,
+  faJs,
+  faGithub,
+  faNodeJs,
+  faReact
+} from '@fortawesome/free-brands-svg-icons';
+import { faDatabase } from '@fortawesome/free-solid-svg-icons';
+import Helmet from 'react-helmet';
+
+import { Spacer } from '../components/helpers';
+import Layout from '../components/layouts/Default';
+import Login from '../components/Header/components/Login';
+
+import './index.css';
+
+const BigCallToAction = () => (
+  <Row>
+    <Col sm={8} smOffset={2} xs={12}>
+      <Login block={true}>Start coding (it's free)</Login>
+    </Col>
+  </Row>
+);
+
+const IndexPage = () => (
+  <Layout disableSettings={true} landingPage={true}>
+    <Helmet>
+      <title>Learn to code | freeCodeCamp.org</title>
+    </Helmet>
+    <Spacer />
+    <Spacer />
+    <Grid className='text-center'>
+      <Row>
+        <h1 className='big-heading'>After Grasshopper, welcome to freeCodeCamp!</h1>
+        <Col>
+          <p className='large-p'>
+          Congratulations, you already have an early experience in programming thanks to the 
+          application Grasshopper! Continue your studies with freeCodeCamp. You will create 
+          beautiful projects, get certifications and it's free!
+          </p>
+        </Col>
+        <Spacer />
+        <Col md={4} sm={12}>
+          <Image
+            alt={
+              'Get great references and connections to start your software ' +
+              'engineer career'
+            }
+            className='landing-icon img-center'
+            responsive={true}
+            src={
+              'https://s3.amazonaws.com/freecodecamp/landing-icon-community.svg'
+            }
+          />
+          <p className='large-p'>
+            Join a supportive community of millions of coders.
+          </p>
+        </Col>
+        <Col md={4} sm={12}>
+          <Image
+            alt='Help nonprofits with bro bono code projects'
```

Comment:
Did you mean to say "pro bono" instead of "bro bono"?

`alt = 'Help non-profits with pro bono code projects'`

Reply:
Yes - it should be "pro bono".